.github/workflows/cli-release.yml

```diff
@@ -0,0 +1,21 @@
+name: cli-release
+
+on:
+  workflow_dispatch:
```

Comment:
This is so that I can run the action manually against PR branches, but I don't believe the final action will have this trigger since we need a tagged release to upload artifacts to.